isthmus/src/main/java/io/substrait/isthmus/SubstraitToSql.java

```diff
@@ -0,0 +1,85 @@
+package io.substrait.isthmus;
+
+import io.substrait.relation.Rel;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.UnaryOperator;
+import org.apache.calcite.config.CalciteConnectionConfig;
+import org.apache.calcite.config.CalciteConnectionProperty;
+import org.apache.calcite.jdbc.CalciteSchema;
+import org.apache.calcite.jdbc.JavaTypeFactoryImpl;
+import org.apache.calcite.plan.Contexts;
+import org.apache.calcite.plan.RelOptCluster;
+import org.apache.calcite.plan.RelOptCostImpl;
+import org.apache.calcite.plan.volcano.VolcanoPlanner;
+import org.apache.calcite.prepare.CalciteCatalogReader;
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.rel.rel2sql.RelToSqlConverter;
+import org.apache.calcite.rel.type.RelDataTypeFactory;
+import org.apache.calcite.rex.RexBuilder;
+import org.apache.calcite.sql.SqlDialect;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlWriterConfig;
+import org.apache.calcite.sql.parser.SqlParseException;
+import org.apache.calcite.sql.validate.SqlValidator;
+import org.apache.calcite.sql2rel.SqlToRelConverter;
+
+public class SubstraitToSql extends SqlConverterBase {
+  private final RelDataTypeFactory factory;
+  private final CalciteCatalogReader catalogReader;
+  private final RelOptCluster relOptCluster;
+  private final SqlValidator validator;
+
+  SubstraitToSql() {
```

Comment:
Can we separate this into two steps? Substrait to Calcite and Calcite to SQL? That way people who want to only use the first part, don't have to reimplement things.

Reply:
I have two steps:  
 - `substraitRelToCalciteRel` : convert from substrait Rel to Calcite Rel
 - toSql : Calcite Rel to string SQL.